src/python/pants/backend/python/target_types.py

```diff
@@ -820,10 +820,12 @@ class PythonTestUtilsGeneratorTarget(TargetFilesGenerator):
     settings_request_cls = PythonFilesGeneratorSettingsRequest
     help = (
         "Generate a `python_source` target for each file in the `sources` field.\n\n"
-        "This target generator is intended for test utility files like `conftest.py`, although it "
-        "behaves identically to the `python_sources` target generator and you can safely use that "
-        "instead. This target only exists to help you better model and keep separate test support "
-        "files vs. production files."
+        "This target generator is intended for test utility files like `conftest.py` or "
+        "`my_test_utils.py`. Technically, it generates `python_source` targets in the exact same "
+        "way as the `python_sources` target generator does, only that the `sources` field has a "
```

Comment:
Why bother explaining this technicality at all? Isn't it an implementation detail?

Reply:
Because our default source globs will not automatically capture `my_test_utils.py`. You have to go out of your way to do that. This paragraph attemps to explain why you might want to do that, but how it's also not strictly necessary.